lib/task-data/base-tasks/generate-tag.js

```diff
@@ -6,9 +6,7 @@ module.exports = {
     friendlyName: 'Generate Tag',
     injectableName: 'Task.Base.Catalog.GenerateTag',
     runJob: 'Job.Catalog.GenerateTag',
-    requiredOptions: [
-        'nodeId'
-    ],
+    requiredOptions: [],
```

Comment:
Oh okay, I had this as non-required so I could have `nodeIds` as null in the definition, while still letting the JSON schema work, but maybe I am misunderstanding the right way to do this.